src/commands/resetlead/ResetUtil.ts

```diff
@@ -0,0 +1,63 @@
+import moment, { Moment } from "moment-timezone";
+
+export enum WvwRegion {
+    EU = "EU",
+    NA = "NA"
+}
+
+function getWvwRegionProperties(wvwRegion: WvwRegion) {
+    let resetWeekDay: number;
+    let resetTimeUTC: number;
+    switch (wvwRegion) {
+        case WvwRegion.EU:
+            resetWeekDay = 5;
+            resetTimeUTC = 18;
+            break
+        case WvwRegion.NA:
+            resetWeekDay = 6
+            resetTimeUTC = 2
+            break
+        default:
+            throw Error("Unmapped WvW Region")
+    }
+    return {resetWeekDay, resetTimeUTC};
+}
+
+export function currentWeek(): number {
+    return moment().utc().isoWeek()
+}
+
+export function getResetForWeek(isoWeek = moment().isoWeek(), year = moment().year(), wvwRegion: WvwRegion = WvwRegion.EU): Moment {
+    let {resetWeekDay, resetTimeUTC} = getWvwRegionProperties(wvwRegion)
+    return moment().tz("UTC")
```

Comment:
Germany being UTC+1, I guess you meant to use `resetTimeUTC` as part of the argument for `tc(...)`?

Reply:
https://github.com/FlussuferOrga/Botgart/pull/114/files/f348b2f9c4be01ba7b2182cde1f50d1f54578e8d#diff-28388770740805ff2a3e136af8f01f773c4b79de6daac8c7a23f15f29a500ac1R240